test/iat.tests.js

```diff
@@ -12,4 +12,13 @@ describe('iat', function () {
     expect(result.exp).to.be.closeTo(iat + expiresIn, 0.2);
   });
 
-});
\ No newline at end of file
+  it('should work with a nbf calculated based on numeric iat', function () {
+    var dateNow = Math.floor(Date.now() / 1000);
+    var iat = dateNow - 30;
+    var notBefore = -50;
+    var token = jwt.sign({foo: 123, iat: iat}, '123', {notBefore: notBefore});
+    var result = jwt.verify(token, '123');
+    expect(result.nbf).to.be.closeTo(iat + notBefore, 0.2);
```

Comment:
since the `timestamp` will be the `iat`, shouldn't it be equal always (instead of "closeTo")

Reply:
I agree completely! I was following the previous test example to keep the style the same, but I will gladly update.

Reply:
Changed to check for direct equals.